Mathlib/Data/Set/Ncard.lean

```diff
@@ -56,47 +447,69 @@ syntax "toFinite_tac" : tactic
 macro_rules
   | `(tactic| toFinite_tac) => `(tactic| apply Set.toFinite)
 
-namespace Set
+syntax "to_encard_tac" : tactic
+
+/-- A tactic useful for transferring proofs for `encard` to their corresponding `card`
+  statements -/
+macro_rules
+  | `(tactic| to_encard_tac) => `(tactic|
+      simp only [←Nat.cast_le (α := ℕ∞), ←Nat.cast_inj (R := ℕ∞), Nat.cast_add, Nat.cast_one])
+
 
 /-- The cardinality of `s : Set α` . Has the junk value `0` if `s` is infinite -/
 noncomputable def ncard (s : Set α) :=
-  Nat.card s
-#align set.ncard Set.ncard
```

Comment:
Here and everywhere else: I think our plan is to leave the `#align`s around for a while (even if they are no longer perfect), so please put them back.